### DISCUSSION TITLE
What is the best possibility to import/export data into/from objects' classification store?

### DISCUSSION THREAD

**Discussion by Blackbam** [pimcore/pimcore#7941] [Q&A / Support]
1) Data Hub: Seems to be great for querying classification store data (https://pimcore.com/docs/data-hub/current/GraphQL/Query/Query_Samples/ClassificationStore.html). Mutation seems to be not possible yet? Or is it just not documented? Is it planned?

2) Grid Import / Export: Seems to be a nice feature. Classification store is not supported though (and probably not easy to implement in a two-dimensional .csv file?)

3) Writing a custom bundle for importing data into classification store via REST API?

4) Unknown possibility?

**Comment by wujku** [comment]
@dvesh3 are you planning to implement grid export for Classification Store attributes?

**Reply by dvesh3** [reply]
AFAIK it already works..

**Reply by wujku** [reply]
When I add Classification Store attribute to grid configuration, it shows "not supported". I use latest version of Pimcore 10.

<img width="441" alt="Zrzut ekranu 2021-07-7 o 16 07 18" src="https://user-images.githubusercontent.com/1288915/124773683-7393d780-df3d-11eb-8852-0aaa037acd26.png">

**Reply by dvesh3** [reply]
when you dragdrop classificationstore, it should open popup to select the key and then you see the data:
![image](https://user-images.githubusercontent.com/5137917/124774533-2ebc7080-df3e-11eb-9c37-a3387c98579a.png)

**Reply by wujku** [reply]
@dvesh3 Yes, I do it the same way.
I close Grid Options by Apply button, opened it again and thats results:
<img width="605" alt="Zrzut ekranu 2021-07-7 o 16 17 31" src="https://user-images.githubusercontent.com/1288915/124775760-2dd80e80-df3f-11eb-9afe-3e392b437f02.png">

The data does not appear, although all lines do have it.

**Reply by dvesh3** [reply]
@wujku please create an issue with reproducible steps. thanks!

**Reply by wujku** [reply]
@dvesh3 thanks for your screenshot. I can probably see what the problem is. Tildes are added in place of dashes.

**Reply by wujku** [reply]
@dvesh3 https://github.com/pimcore/pimcore/issues/9702

**Comment by dpfaffenbauer** [comment]
DataDefinitions support import and export

**Reply by Blackbam** [reply]
@dpfaffenbauer Still with DataHub the mutation for classification store is not yet possible right?

**Reply by dpfaffenbauer** [reply]
dont know
